codebuild/bin/cppcheck_suppressions.txt

```diff
@@ -9,6 +9,56 @@ ConfigurationNotChecked:tls/s2n_x509_validator.c
 ConfigurationNotChecked:utils/s2n_safety.c
 ConfigurationNotChecked:utils/s2n_socket.c
 
+// cppcheck Message: [pq-crypto/bike_r1/bike_defs.h:85]: (information:ConfigurationNotChecked) Skipping configuration 'static_assert' since the value of 'static_assert' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly.
+// Reason: Same as above.
+ConfigurationNotChecked:pq-crypto/bike_r1/bike_defs.h
+ConfigurationNotChecked:pq-crypto/bike_r1/gf2x_mul.c
+ConfigurationNotChecked:pq-crypto/bike_r1/sha.h
+ConfigurationNotChecked:pq-crypto/bike_r1/aes_ctr_prf.c
+ConfigurationNotChecked:pq-crypto/bike_r1/secure_decode_portable.c
+ConfigurationNotChecked:pq-crypto/bike_r1/decode.c
+ConfigurationNotChecked:pq-crypto/bike_r2/bike_defs.h
+ConfigurationNotChecked:pq-crypto/bike_r2/gf2x_mul.c
+ConfigurationNotChecked:pq-crypto/bike_r2/sha.h
+ConfigurationNotChecked:pq-crypto/bike_r2/aes_ctr_prf.c
+ConfigurationNotChecked:pq-crypto/bike_r2/secure_decode_portable.c
+ConfigurationNotChecked:pq-crypto/bike_r2/decode.c
+ConfigurationNotChecked:pq-crypto/bike_r2/bike_r2_kem.c
+
 // cppcheck Message: [tests/unit/s2n_timer_test.c:49]: (style:redundantAssignment) Variable 'mock_time' is reassigned a value before the old one has been used.
 // Reason: s2n_config_set_monotonic_clock() takes a reference to mock_time so that whenever it's modified locally, the timer sees the update when it dereferences the pointer.
 redundantAssignment:tests/unit/s2n_timer_test.c
+
+// cppcheck Message: [pq-crypto/sike_r2/fp.c:24]: (error:shiftTooManyBits) Shifting 32-bit value by 64 bits is undefined behaviour.
+// Reason: This is actually a 128-bit typedef being shifted by 64 bits.
+shiftTooManyBits:pq-crypto/sike_r2/fp.c
+shiftTooManyBits:pq-crypto/sike_r2/fpx.c
```

Comment:
We probably shouldn't put line numbers in these comments in case they change. If the suppression only applies to one line, it should be done in-line in the C file.

Reply:
I removed the specific line numbers from the messages, and moved suppressions in-line that made sense.